perun-base/src/main/resources/perun-roles.yml

```diff
@@ -6653,99 +6675,137 @@ perun_policies:
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   vo-addMail_ApplicationForm_ApplicationMail_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   group-deleteMailById_ApplicationForm_Integer_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   vo-deleteMailById_ApplicationForm_Integer_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   copyMailsFromVoToVo_Vo_Vo_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   copyMailsFromVoToGroup_Vo_Group_boolean_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   copyMailsFromGroupToGroup_Group_Group_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   group-sendMessage_Application_MailType_String_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   vo-sendMessage_Application_MailType_String_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   group-sendInvitation_Vo_Group_String_String_String_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   vo-sendInvitation_Vo_Group_String_String_String_policy:
     policy_roles:
       - VOADMIN: Vo
       - TOPGROUPCREATOR: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   group-sendInvitation_Vo_Group_User_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   vo-sendInvitation_Vo_Group_User_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   #RegistrarManagerImpl
   createApplicationFormInVo_Vo_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   createApplicationFormInGroup_Group_policy:
     policy_roles:
       - GROUPADMIN: Group
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
```

Comment:
Are we really changing mentioned Vo or Group? Maybe general MFA would be better, since only empty application form is created and no attribute values are set, or?

Reply:
We discussed this, and global MFA would enforce MFA for all forms, this way only for the groups / Vos marked as critical.